src/dodal/devices/i18/diode.py

```diff
@@ -1,8 +1,35 @@
-from ophyd_async.core import (
-    StandardReadable,
-)
+from ophyd_async.core import StandardReadable, StrictEnum
 from ophyd_async.epics.core import epics_signal_r
 
+from dodal.devices.positioner import create_positioner
+
+
+class FilterAValues(StrictEnum):
+    """Maps from a short usable name to the string name in EPICS"""
+
+    AL_2MM = "2 mm Al"
+    AL_1_5MM = "1.5 mm Al"
+    AL_1_25MM = "1.25 mm Al"
+    AL_0_8MM = "0.8 mm Al"
+    AL_0_55MM = "0.55 mm Al"
+    AL_0_5MM = "0.5 mm Al"
+    AL_0_3MM = "0.3 mm Al"
+    AL_0_25MM = "0.25 mm Al"
+    AL_0_15MM = "0.15 mm Al"
+    AL_0_1MM = "0.1 mm Al"
+    AL_0_025MM = "0.025 mm Al"
+    AL_GAP = "Gap"
```

Comment:
```suggestion
    AL_0_025MM = "0.025 mm Al"
    AL_0_05MM = "0.05 mm Al"
    AL_GAP = "Gap"
```

must: missing this enum